deployments/docker/src/keycloak.docker

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/keycloak/keycloak:22.0.1 as keycloak-builder
+
+WORKDIR /opt/keycloak
+# note: for development set up early; use proper certification in production.
+RUN keytool -genkeypair -storepass password -storetype PKCS12 -keyalg RSA -keysize 2048 \
+	-dname "CN=server" -alias server -ext "SAN:c=DNS:localhost,IP:127.0.0.1" \
```

Comment:
should the server on  line 6 be hard coded  or read from yaml file?

Reply:
nah, this isn't really used. This sets up keys used for the HTTPS cert, but we're enabling and using HTTP (specifically to avoid having to deal with certs in the test deployment). There isn't really a way to disable HTTPS (that I've seen), so the keys need to generated even if they're not used.